node/src/command.rs

```diff
@@ -1,11 +1,9 @@
 use crate::{
     chain_spec,
-    cli::{
-        Cli,
-        RelayChainCli,
-        Subcommand,
-    },
+    cli::{Cli, RelayChainCli, Subcommand},
+    service::{new_partial, DataHighwayRuntimeExecutor}
 };
+use datahighway_runtime::RuntimeApi;
```

Comment:
just wondering if we should rename this to `datahighway_parachain_runtime` or similar throughout the codebase? (to differentiate it from the standalone chain)